stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -65,6 +65,13 @@ function nnz(x::SparseColumnView)
     return length(nzrange(parent(x), colidx))
 end
 
+
+function Base.sizehint!(v::SparseVector, newlen::Integer)
```

Comment:
This seems a little strange to me.  This points the way towards a corresponding `sizehint(::SparseMatrixCSC, nstored)`, which is even odder.

Perhaps this should be its own function that `SparseArrays` exports, like `storedhint!`?

Reply:
I think `sizehint!` works for this: it's the storage size of the vector/matrix. Yes, that's not the same as the vector size, but I think that's ok—`sizehint!` is inherently about storage, after all.